pandas/tests/io/json/test_normalize.py

```diff
@@ -66,6 +66,25 @@ def author_missing_data():
          }]
 
 
+@pytest.fixture
+def address_missing_data():
```

Comment:
I think calling this `missing_meta_data` would make it clearer what the purpose of the fixture is

Reply:
Will do